guides/common/modules/ref_customizing-the-registration-templates.adoc

```diff
@@ -21,9 +21,12 @@ You can configure the following global parameters by navigating to *Configure* >
 
 ifndef::satellite[]
 * The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `false`.
+Set the parameter type to `boolean` to override the parameter value.
 endif::[]
 ifdef::satellite[]
 * The `host_registration_insights` parameter is used in the `insights` snippet, the default value is `true`.
+Set the parameter type to `boolean` to override the parameter value.
+If the parameter is set to `false`, it prevents hosts from appearing in the {Team} Inventory reports and also the `insights-client` from uploading reports to `rh-cloud`.
```

Comment:
```suggestion
If the parameter is set to `false`, it prevents hosts from appearing in the {Team} Inventory reports and also the Insights client from uploading reports to RH Cloud.
```
I'm suggesting a couple of formatting improvements. "RH Cloud" is found in the WebUI, so I think it's vital to spell it like this.
Again, I suggest taking this line out of the condition, because it's relevant for all build targets.

Reply:
@Lennonka We are discussing [here](https://github.com/theforeman/foreman-documentation/pull/2180#discussion_r1187306218) the potential possibility of adding attributes for RH Cloud. Should we wait for it or we shall move forward here?

Reply:
@mjivraja Perhaps we should introduce a new attribute for "Red Hat Cloud" as well in another PR so that we start using the product name consistently? #2180 is about something different if I understand correctly.

Reply:
In this [PR](https://github.com/theforeman/foreman-documentation/pull/2180#pullrequestreview-1417989076) Maximilian shares sensible data to avoid introducing the new attribute for RH Cloud. 

However, I also agree with your purpose to introduce the attribute for "Red Hat Cloud" to start using the product name consistently.

Reply:
@mjivraja Yes, but he's talking about an attribute for all builds and for a program name in the CLI. We can add an attribute for the product title only for the Satellite build.